ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -0,0 +1,117 @@
+# Interface Change Log
```

Comment:
@Jasagredo said: 

> I am okey with the idea of the ChangeLog although I would suggest a couple of improvements:
>
> - Keep links to code/PRs as changes can span several PRs.
> - Allow appending information to an entry to further refine it. If something is fundamentally wrong, probably just strikethrough it and add a reference to some explanation below, instead of deleting it.

- Hmm. I think we should have one PR per log entry. We don't merge a PR until it builds, so at the very least some coherent portion of the ultimate change will already be in place. The log entry for each preparatory PR can at least list what changed and then defer more details by explicitly saying "a subsequent log entry will say more". How disruptive/challenging do you think that scheme would be for us?
- If we add the `<s>` and `</s>` tags on their own line, then we can strikethrough the surrounded lines without altering them, thereby preserving the `git blame` output 👍. I tested this a bit and if even works for surrounded headers as long as you add blanks lines around the strikethrough tags.